src/diagnostics/riscv64/disasm-riscv64.cc

```diff
@@ -1629,6 +1645,16 @@ void Decoder::DecodeCIType(Instruction* instr) {
   }
 }
 
+void Decoder::DecodeCIWType(Instruction* instr) {
+  switch(instr->RvcOpcode()) {
+    case RO_C_ADDI4SPN:
+      Format(instr, "addi       'Crs2s, sp, 'Cimm8Addi4spn");
```

Comment:
It looks like the other C instructions do the same, but I am curious why we disassemble as the full instruction instead of showing the C instruction.

Reply:
Discussed at meeting.